guide/programs_that_come_with_dart.rst

```diff
@@ -0,0 +1,124 @@
+
+Programs included with DART
+===========================
+
+
+DART comes with a collection of programs that can be used in
+various ways when doing data assimilation tasks.  
+
+
+Assimilation
+------------
+
+*filter* 
+   `main program <../assimilation_code/programs/filter/filter.rst>`__  which performs data assimilation
+
+
+
+OSE or OSSE - simulation experiments
+------------------------------------
+
+*perfect_model_obs* 
+   generates `synthetic observations from a single model state <../assimilation_code/programs/perfect_model_obs/perfect_model_obs.rst>`__
+
+*create_obs_seq* 
+   generates `synthetic observations of any kind <../assimilation_code/programs/create_obs_seq/create_obs_seq.rst>`__
+
+*create_fixed_network_seq* 
+   generates `a time sequence of observations <../assimilation_code/programs/create_fixed_network_seq/create_fixed_network_seq.rst>`__
+
+
+
+Diagnostics
+-----------
+
+*obs_diag* 
+   `main diagnostic <../assimilation_code/programs/obs_diag/obs_diag.rst>`__ program which generates summary statistics for bins of obs
+
+*obs_seq_to_netcdf* 
+   converts DART's `obs_seq format to netcdf <../assimilation_code/programs/obs_seq_to_netcdf/obs_seq_to_netcdf.rst>`__
```

Comment:
Sorry ... all of the links that end in .rst are broken on readthedocs.

Since this is in the guide directory, I suspect its main purpose will be for people to read them using readthedocs - true? If that is the case, they should be of the form

:doc:`string you want displayed <relative_path_to_file_with_no_file_extension>`

I'm happy to change them ... just want to know what the intended discovery mechanism is ...
Or do I just let this slide because of PR 199 ...

Reply:
@kdraeder     use what you can from this  and put it in the   programs/  file in your pull request 199 ish

Reply:
@kdraeder    and then delete the 'programs_that_come_with_dart.rst' ...

Reply:
file has been deleted.